monkey/infection_monkey/model/__init__.py

```diff
@@ -41,3 +41,9 @@
                     "&&  %(monkey_path)s %(monkey_type)s %(parameters)s"
 
 DOWNLOAD_TIMEOUT = 300
+
+
+# Commands needed for PE
+
+REMOVE_LASTLINE = "sudo sed -i '$ d' %(file_name)s"
+ADDUSER_TO_SUDOERS = "echo '%(user_name)s ALL = NOPASSWD: ALL' | sudo tee -a /etc/sudoers"
```

Comment:
THB I don't know what's up with this file. I think this file started as "lets put common commands of dropper and monkey on the same file" and ended up being "lets cram everything here until we have a huge file of unrelated commands". If you think that these commands will be re-used throughout PE's - define them in actions/__init__.py and all PE classes will have access to them. If they're specific to one technique - define them in corresponding PE file.

Reply:
Sure, I was meaning to move this to  models.py or init.py under pe/actions. Thanks for reminding me.